src/View/Components/Input.php

```diff
@@ -55,7 +55,7 @@ public function render(): View|Closure|string
                         <span>
                             {{ $label }}
 
-                            @if($attributes->has('required'))
+                            @if($attributes->has('required') && $attributes->get('required'))
```

Comment:
If you only keep `$attributes->get('required')` will work?

Reply:
this actually works, yes